godot/src/test/avatar/avatar_tester_standalone.tscn

```diff
@@ -1,7 +1,8 @@
-[gd_scene load_steps=5 format=3 uid="uid://4227l72a8kdu"]
+[gd_scene load_steps=6 format=3 uid="uid://4227l72a8kdu"]
 
 [ext_resource type="PackedScene" uid="uid://cy43hvd5bivso" path="res://src/ui/components/backpack/avatar_preview.tscn" id="1_1eenv"]
 [ext_resource type="Script" path="res://src/test/avatar/avatar_tester_standalone.gd" id="1_l1th6"]
+[ext_resource type="Environment" uid="uid://c6yftsnvufsg8" path="res://assets/sky/cli_environment.tres" id="3_2n2br"]
```

Comment:
we are taking screenshots for the Profile using the AvatarPreview, why the environment is different from the environment for the Avatar Tester and the Avatar Preview?

Reply:
Answered on Discord, for the record: We are going to do this way for the CI. We can look at this on the future in the Visual Improvements